python/test/unit/fem/test_expression.py

```diff
@@ -251,61 +357,3 @@ def e_exact(x):
             x = mesh.geometry.cmap.push_forward(quadrature_points, xg)
             e_exact_eval[Q_dofs_unrolled[cell]] = e_exact(x.T).T.flatten()
         assert np.allclose(local.array, e_exact_eval)
-
-
-@pytest.mark.skipif(np.issubdtype(PETSc.ScalarType, np.complexfloating),
```

Comment:
I think this test should be added back in the final PR request.

Reply:
It would be better to include this into a demo. I agree it demonstrates useful technique, but it is not a test of untested code path itself, right?

Reply:
The code path is already tested, that is true.
There are quite a lot of things we should put in demos, as already discussed earlier in this PR.
I guess we need to make a plan of what to show where, as currently, expression is only used in 
elasticity demo, and it does not really highlight any use-case, as it only evaluates the von mises stresses on the reference element (without a pullback from the physical coordinate system as done in this test).
The demo should either do as in this test, or use interpolate and put it in a suitable function space.